tests/src/Functional/PublicationPageNavigationTest.php

```diff
@@ -91,8 +91,8 @@ public function testPreviousNextLinks() {
 
     $this->drupalLogin($adminUser);
     $this->drupalGet('/node/' . $node_child_one->id());
-    $this->assertSession()->responseContains('<a href="/publication-parent-page" rel="prev" title="Go to previous page">');
```

Comment:
The test as-is checks the links go to the right place. The updated version only checks that links are present, which feels like a step backwards.  I think the best fix for this would be to use xpath to find the right <a> tags, then check their href attribute is what we expect.

Have you got a link to the failing run so I can get an idea what change broke the test?

Reply:
Yep it's on the 1.x branch

Reply:
You're right. It's this: https://github.com/localgovdrupal/localgov_base/pull/596